src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.tsx

```diff
@@ -74,6 +85,23 @@ const VisualRuleForm: React.FC<IVisualRuleFormProps> = (props) => {
         theme: theme
     });
     const commonFormStyles = getPanelFormStyles(theme, rootHeight);
+    const typeOptions: Array<IDropdownOption> = useMemo(
+        () =>
```

Comment:
this seems like a copy paste from somehwere, does it makes sense to have a utility for it?